.github/actions/deploy-conf/README.md

```diff
@@ -0,0 +1,30 @@
+# deploy-conf Shared GitHub Action
+The `deploy-conf` is a parameterised reusable GitHub action that deploys of CHT applications to a running CHT instance.
+
+## CHT App Requirements
+* cht-conf@3.7.0 or above
+
+## Example GitHub Step
+
+```
+name: Example GitHub Workflow yml
+
+on: ['deployment']
+
+jobs:
+  deployment:
+    runs-on: ubuntu-22.04
```

Comment:
i think we use `runs-on: ubuntu-latest` for most deployment scripts now.  we might consider that  here also so this doc doesn't need upkeep

Reply:
Thanks, done!